### ISSUE TITLE
Icon: add fill color as bookmark is already present

### ISSUE THREAD

**Issue by tobiasnix** [damko/freedommarks-browser-webextension#20]
Would be nice if the icon got filled with a color on sides that are already bookmarked (like the start in firefox: filled - already bookmarked, empty - not bookmarked).

**Comment by damko** [comment]
ABSOLUTELY! this is a feature that was already implemented but I had to remove it because of the changes that happened on the API side. Not sure if atm this is possible again.

**Comment by IzzySoft** [comment]
Speaking of colors and fill: while on Firefox the FM icon was black and easy to see, on Waterfox it is almos white – and thus hard to spot on the light-gray bar. Anything that can be done about that?

![Selection_106](https://user-images.githubusercontent.com/6781438/54957832-35cd3980-4f54-11e9-8faf-d6e14dc4ab13.png)
